src/analyses/constant_propagator.cpp

```diff
@@ -331,6 +331,29 @@ void constant_propagator_domaint::assign(
   dest.set_to(lhs, rhs);
 }
 
+/*******************************************************************    \
```

Comment:
That final `\` seems way off. (I'm being picky about comments to make sure automated conversion works.)